frontend/cypress/integration/queryBuilder.spec.js

```diff
@@ -45,6 +45,19 @@ describe('The Query Builder', () => {
     cy.get('.invalid-feedback').should('not.be.visible');
   });
 
+  it('that encountered an error or interruption during alias creation shows a warning', () => {
+    // Mock datasets route
+    cy.fixture('datasets').then((datasets) => {
+      cy.intercept('api/datasets/mine/', datasets);
+    });
+
+    cy.visit('/');
+    cy.get('.p-0 div a').first().click({ force: true });
+    cy.get('.p-1').contains(
```

Comment:
These classes `p-0, p-1` are at high risk of changing, which will break your tests. I'd rather you add a testId to your elements.